tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -580,7 +580,7 @@ static PyObject * THPVariable_tolist(PyObject* self, PyObject* args)
   HANDLE_TH_ERRORS
   jit::tracer::warn("Converting a tensor to a Python list", jit::tracer::WARN_PYTHON_DATAFLOW);
   auto self_ = reinterpret_cast<THPVariable*>(self)->cdata;
-  return torch::utils::tensor_to_list(self_.data());
+  return torch::utils::tensor_to_list(self_);
```

Comment:
It's okay to use `self_` instead of `self_.data()` here, because the implementation of `torch::utils::tensor_to_list(...)` https://github.com/pytorch/pytorch/blob/9b272affdebf2392d8ede7f83223bf30392a6e43/torch/csrc/utils/tensor_list.cpp#L31-L41 doesn't care whether the tensor requires grad or not.